pages/index.tsx

```diff
@@ -62,10 +66,40 @@ export default function Home() {
             ))}
           </ButtonGroup>
         </SectionContainer>
+        <SectionContainer sx={{ margin: 'auto', maxWidth: '65rem' }} maxWidth={false}>
+          <Box>
+            <Grid container>
+              <Grid item xs={12}>
+                <Typography variant="h2" sx={{ textAlign: 'center' }}>
+                  {content.ourMission.header}
+                </Typography>
+              </Grid>
+              <Grid item sm={12} md={6}>
+                <ImageContainer
+                  src={content.ourMission.imgsrc as string}
+                  alt=""
+                  width={406}
+                  height={306}
+                  style={{ width: '100%' }}
+                  useImageDimensions
+                />
+              </Grid>
+              <Grid item sm={12} md={6}>
+                <Typography variant="subtitle1" sx={{ mt: '4.375rem', textAlign: 'center' }}>
+                  <MuiMarkdown>{content.ourMission.body}</MuiMarkdown>
+                </Typography>
+
+              </Grid>
+            </Grid>
+          </Box>
+        </SectionContainer>
+
         {content.gridSections.map((section) => (
           <SectionContainer key={section.id} id={section.id}>
             <PaperSection
               title={section.title}
+              subtitle={section.subtitle}
+              // added subtitle above
```

Comment:
Let's try and keep comments only for explaining something that might be difficult to understand. When we review the pull requests, everything you have added, changed, or deleted is highlighted so we don't need to add these.

Reply:
Removed comment